generators/client/generator.mjs

```diff
@@ -112,6 +112,22 @@ export default class JHipsterClientGenerator extends BaseApplicationGenerator {
           this.jhipsterConfig.clientFramework = ANGULAR;
         }
       },
+
+      configureDevServerPort() {
```

Comment:
should be located on `./support/whateverfile` or at two differents location (angular and client)